internal/pkg/config/defaults/lte10000_limits.yml

```diff
@@ -53,3 +53,7 @@ server_limits:
     interval: 1ms
     burst: 2000
     max: 4000
+  audit_unenroll_limit:
```

Comment:
The default/env limits for the audit/unenroll endpoint are set to the same values as the ack endpoint (excluding body which is set to 1 kb)

Reply:
+1